_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,14 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Host registration improvements
```

Comment:
Due to the manual branching this change should be moved to `_includes/manuals/2.5/1.2_release_notes.md`